src/touch.c

```diff
@@ -11,11 +11,7 @@
 uint32_t sent_time;
 uint32_t delta;
```

Comment:
This should actually not be "delta" (difference between old and new), but it is the current value of the time it takes